docs/api/advanced/snapshot.mdx

```diff
@@ -7,22 +7,89 @@ description: 'Create a snapshot of current state'
 
 # snapshot
 
-#### A snapshot takes a proxy and returns a immutable object, unwrapped from the proxy.
```

Comment:
The sentence seemed really long for a `####` header, so just removed the header notation...